### PR TITLE
[#182] Fix Load DB Dump test

### LINKED ISSUE #182: Fix Load DB Dump test

### PR DESCRIPTION
**Pull request by JanKapala**
Closes #182

### CODE REVIEW

File: tests/tasks/test_db.py [hunk 1]
```diff
@@ -25,7 +25,7 @@ def test_handle_db_dump(mongo, mp_dump_data):
             {
                 field: value
                 for field, value in mongo_model_to_json(x).items()
-                if field not in ("dataframe", "tensor", "synthetic")
+                if field not in ("dataframe", "tensor", "embedded_tensor", "synthetic")
```
Comment (wujuu) [inline tests/tasks/test_db.py:28]: Maybe add a constant tuple - FIELDS_NOT_INLCLUDED_IN_MP

**Review by wujuu — CHANGES_REQUESTED** [review]

File: tests/tasks/test_db.py (hunk 1 above)
Comment (wujuu) [reply tests/tasks/test_db.py:28]: Makes it more understandable why we are not checking for these fields

Comment (JanKapala) [reply tests/tasks/test_db.py]: good idea, added

File: tests/tasks/test_db.py [hunk 2]
```diff
@@ -8,6 +8,7 @@
 from tests.factories.user_action import UserActionFactory
 from tests.helpers import mongo_model_to_json
 
+FIELDS_NOT_INLCLUDED_IN_MP = ("dataframe", "tensor", "embedded_tensor", "synthetic")
```
Comment (wujuu) [inline tests/tasks/test_db.py:11]: sorry, typo in INCLUDED

**Review by wujuu — CHANGES_REQUESTED** [review]

File: tests/tasks/test_db.py (hunk 2 above)
Comment (JanKapala) [reply tests/tasks/test_db.py]: XD fixed

**Review by wujuu — APPROVED** [review]

**Pull request merged**